### PR TITLE
Fixed issue #1124

### PR DESCRIPTION
**Pull request by ghost**
Fixed drop-down menu. The description can be found here.

 issue https://github.com/python/pythondotorg/issues/1124

### CODE REVIEW

**Review by berkerpeksag — CHANGES_REQUESTED** [review]
Thank you for the PR. `mq.css` is an automatically generated file. Please modify `static/sass/mq.scss` instead.

File: static/sass/mq.css [hunk 1]
```diff
@@ -2528,3 +2528,11 @@ html[xmlns] .slides { display: block; }
 
 @viewport {
   width: device-width; }
+
+
+.element-4 ul,
+.element-2 ul{
+  height:288px;
```
Comment (berkerpeksag) [inline static/sass/mq.css:2535]: We don't use px as unit. Please use the `px2em` helper.

Also, is there a way to fix this without set `<ul>`'s height to a fixed value?

File: static/sass/mq.css [hunk 2]
```diff
@@ -2528,3 +2528,11 @@ html[xmlns] .slides { display: block; }
 
 @viewport {
   width: device-width; }
+
+
+.element-4 ul,
```
Comment (berkerpeksag) [inline static/sass/mq.css:2533]: We support multiple devices so this should be moved into one of the applicable media blocks like

```scss
@media (min-width: px2em( ... )) {

}
```

**Review by berkerpeksag — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
I have made changes and made a pull request, but it has gone to the new pull request.
Link: https://github.com/python/pythondotorg/pull/1209

**Comment by berkerpeksag** [comment]
Superseded by #1209.

**Pull request closed without merging**